kitsune/search/admin.py

```diff
@@ -48,6 +38,33 @@ class DeleteError(Exception):
     pass
 
 
+def create_batch_id():
+    """Returns a batch_id"""
+    # TODO: This is silly, but it's a good enough way to distinguish
+    # between batches by looking at a Record. This is just over the
+    # number of seconds in a day.
+    return str(int(time.time()))[-6:]
+
+
+def handle_reconcile(request):
+    """Reconcile all the things"""
+    outstanding = Record.objects.outstanding().count()
+    if outstanding > 0:
+        raise ReindexError('There are %s outstanding chunks.' % outstanding)
+
+    batch_id = create_batch_id()
+
+    for cls, indexable in get_indexable():
+        index = cls.get_index()
+        doc_type = cls.get_mapping_type_name()
+        chunk_name = 'Reconciling: %s' % doc_type
+        rec = Record(batch_id=batch_id, name=chunk_name)
+        rec.save()
+        reconcile_task.delay(index, batch_id, rec.id, doc_type)
+
+    return HttpResponseRedirect(request.path)
```

Comment:
This is new. I figure it might be helpful to be able to reconcile the db vs. the index all by itself. Having said that, it timed out on my machine when I tried. I suspect it's because it's pulling the ids for too many things from ES and thus it won't ever work. If that's true, we might want to either change it to chunkify things or nix it if we don't care.